keyboards/helixlab/aaru65/hotswap/keymaps/default/keymap.c

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 luantty2 (@luantty2)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+#ifdef FADER_ENABLE
+#    include "fader.h"
+#endif
+#include "debug_list.h"
```

Comment:
These should be moved to a aaru64.h file, which will be what is used for `QMK_KEYBOARD_H`. 

```suggestion
```